diff --git a/src/ckernels/grayskull/common/inc/ckernel_sfpu.h b/src/ckernels/grayskull/common/inc/ckernel_sfpu.h
index 8cabd8ca7..f0d241ddb 100644
--- a/src/ckernels/grayskull/common/inc/ckernel_sfpu.h
+++ b/src/ckernels/grayskull/common/inc/ckernel_sfpu.h
@@ -62,6 +62,8 @@ sfpi_inline vFloat sfpu_reciprocal(const vFloat in)
 {
     vInt orig_exp;
 
+    for (;;);
+
     if constexpr (max_iter == 1) {
         // If we are only doing one iteration of the MAD loop, then we only need to use one LREG for the MAD instructions because we have our "first guess" in a hard-coded register
         // This allows us to avoid having to load back in the original value later on
